sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawNetwork.java

```diff
@@ -3,30 +3,31 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
-import java.util.Map;
+import java.util.stream.Collectors;
 
 /**
  * Raw YAML network.
  */
 public class RawNetwork {
     private final List<Integer> hostPorts;
     private final List<Integer> containerPorts;
-    private final Map<String, String> labels;
+    private final String labelsCsv;
 
     @JsonCreator
     private RawNetwork(
             @JsonProperty("host-ports") List<Integer> hostPorts,
```

Comment:
We obviously aren't using this yet, so consider this non-applicable future commentary for the purposes of this PR.

Aren't these ports for doing mapping? Why are they separate lists?